ember-cli/2016-05/may-26.md

```diff
@@ -0,0 +1,38 @@
+# ember-cli Meeting Notes - 2016-05-26
+
+## Attendees
+
+- Nathan Hammond
+- Miguel Madero
+- Tracy Lee
+- Kelly Selden
+- Jake Bixby
+- Tobias Bieniek
+
+## Actions
+
+- [x] Nathan: Release 2.6.0-beta3
+
+## Discussion Notes
+
+### core-object
+
+- Stef did go through and fix it.
```

Comment:
Accurate as of the date of the meeting. Not sure if work is complete on it, but that's okay.